src/hotspot/share/logging/logTag.hpp

```diff
@@ -160,7 +160,6 @@ class outputStream;
   LOG_TAG(preorder)  /* Trace all classes loaded in order referenced (not loaded) */ \
   LOG_TAG(preview)   /* Trace loading of preview feature types */ \
   LOG_TAG(promotion) \
-  LOG_TAG(protectiondomain) /* "Trace protection domain verification" */ \
```

Comment:
Not 100% sure about this. We don't really have a policy for "deprecating" or removing log tags. I think it unlikely anyone enables this logging "just because", so it seems okay for this case.

Reply:
Given that I'm probably the only one that has ever used this tag (or maybe also Ioi), I think it's safe to remove.